anitya/lib/backends/__init__.py

```diff
@@ -30,82 +30,9 @@
 import anitya
 import anitya.app
 from anitya.lib.exceptions import AnityaPluginException
+from anitya.lib.versions import RpmVersion
 import six
 
-try:
-    from rpm import labelCompare as _compare_rpm_labels
-except ImportError:
-    # Emulate RPM field comparisons as described in
-    # http://stackoverflow.com/questions/3206319/how-do-i-compare-rpm-versions-in-python/3206477#3206477
```

Comment:
@ncoghlan actually implemented this based on the description in the referenced answer (which in turn is a summary of the RPM C implementation) and kindly posted his code to the SO answer. I don't think that's problematic.